### PR TITLE
complete chinese translation

### PR DESCRIPTION
**Pull request by wgzhao**

### CODE REVIEW

File: caravel/models.py [hunk 1]
```diff
@@ -438,7 +438,8 @@ def dttm_converter(self, dttm):
             'oracle':
                 """TO_TIMESTAMP('{}', 'YYYY-MM-DD"T"HH24:MI:SS.ff6')""".format(
                     dttm.isoformat()),
-            'presto': default,
+            'presto': "CAST('{}' AS TIMESTAMP)".format(
```
Comment (xrmx) [inline caravel/models.py]: This looks unrelated

File: caravel/config.py [hunk 2]
```diff
@@ -113,7 +113,7 @@
 LANGUAGES = {
     'en': {'flag': 'us', 'name': 'English'},
     # 'fr': {'flag': 'fr', 'name': 'French'},
-    # 'zh': {'flag': 'cn', 'name': 'Chinese'},
+    'zh': {'flag': 'cn', 'name': 'Chinese'},
```
Comment (mistercrunch) [inline caravel/config.py]: Is the website mostly translated at this point? I feel like we may want to leave the default to `en` only until we have really good translation coverage.

**Comment by mistercrunch** [comment]
Also I'm not quite sure why this broke the build... maybe rebasing would help?

File: caravel/config.py (hunk 2 above)
Comment (wgzhao) [inline caravel/config.py]: yes. I try to translate which include mako template ,but also almost python files . for example .
`caravel/forms.py` 

```python
viz_type': SelectField(
                'Viz',
                default='table',
                choices=[(k, v.verbose_name) for k, v in viz_types.items()],
                description=_("The type of visualization to display")),
            'metrics': SelectMultipleSortableField(
                _('Metrics'), choices=datasource.metrics_combo,
                default=[default_metric],
                description=_("One or many metrics to display")),
            'metric': SelectField(
                _('Metric'), choices=datasource.metrics_combo,
                default=default_metric,
                description="Chose the metric"),
            'stacked_style': SelectField(
                _('Chart Style'), choices=self.choicify(
```

I fork the project and create `i18n` branch . I will send a new PR ASP

**Comment by mistercrunch** [comment]
Build fails. Please re-open a clean PR when ready.
